docs/source/index.rst

```diff
@@ -25,8 +25,9 @@ Guide
    databases
    running-autometa
    examining-results
-   Scripts/scripts
+   scripts/Usage
```

Comment:
Yes, I have yet not been able to integrate `parse_argprase.py` with `conf.py`. It can be done to by making the whole `parse_argparse.py` script as a function in `conf.py` but I was thinking of making `parse_argparse.py` as a module and importing it.

Reply:
That's probably a good route to take. Sounds like this will be easily resolved once the `parse_argparse.py` script is ready.